<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Selector</class>
 <widget class="QWidget" name="Selector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>316</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>控件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_selectorContral">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_SelectorContral">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>*</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Widget</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PushButton</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RadioButton</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>toolBox</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TabWidget</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ComboBox</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LineEdit</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TextEdit</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SpinBox</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DoubleSpinBox</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TimeEdit</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DataEdit</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Data/TimeEdit</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dial</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Silder</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Laber</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LCDNumber</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ProgressBar</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioButton_selectorFutrue">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>后代</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_selectorFutrue">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="comboBox_selectorFutrue">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按钮</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="radioButton_selectorChild">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>子类</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_selectorChild">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="comboBox_selectorChild">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按钮</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>属性</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_selectorattribute">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>值</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_selectorattribute">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QCheckBox" name="checkBox_selectClassMatch">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>类型匹配</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_selectorID_Match">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>ID匹配</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>子控件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_selectorChildContral">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>伪状态</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_selectorFakeStatus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QDialogButtonBox" name="buttonBox_selector">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
